src/test/groovy/io/codearte/jfairy/producer/company/locale/CompanyProviderSpec.groovy

```diff
@@ -0,0 +1,82 @@
+package io.codearte.jfairy.producer.company.locale
+
+import io.codearte.jfairy.data.DataMaster
+import io.codearte.jfairy.producer.BaseProducer
+import io.codearte.jfairy.producer.VATIdentificationNumberProvider
+import io.codearte.jfairy.producer.company.CompanyProvider
+import spock.lang.Specification
+
+import static io.codearte.jfairy.producer.company.CompanyProvider.*
+
+class CompanyProviderSpec extends Specification {
+
+    def baseProducer = Mock(BaseProducer)
+    def dataMaster = Mock(DataMaster)
+    def vatIdentificationNumberProvider = Mock(VATIdentificationNumberProvider)
+
+    def "should generate should remove slashes in company email"() {
```

Comment:
The name of this test should be modified (too many shoulds - select one thing you are testing)